fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,44 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on label `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    the event corresponding to `y=1`.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to `y = 1` in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values of the sensitive feature, multiplied by
+    the number of unique non-NaN values of the constructed `event` array,
+    whose entries are either NaN or `label=1` (so only one unique non-NaN value), multiplied
+    by two (for the Lagrange multipliers for positive and negative constraints).
+
+    With these definitions, the :meth:`signed_weights` method
+    will calculate the costs like how it is calculated in Example 4 of
+    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_ but for all
+    `y=1' samples and subsequently, add a weight of `0` for
+    all other samples (for all `y=0`).
```

Comment:
Replace the part of the comment that starts with "but for all" with:
```
   , but will only use the weights equal to zero for `y=0`.
```

Reply:
```
    With these definitions, the :meth:`signed_weights` method
    will calculate the costs like how it is calculated in Example 4 of
    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_  for all
    `y=1' samples, but will use the weights equal to zero for `y=0`.
```
Is this better?

Reply:
how about this:
```
    With these definitions, the :meth:`signed_weights` method
    will calculate the costs for `y=1` as they are calculated in Example 4 of
    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_,
    but will use the weights equal to zero for `y=0`.
```

Reply:
Yes, I think this makes the comment more concise. Thank you!